src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -14,7 +14,7 @@ namespace FakeItEasy.Core
     /// </summary>
     public partial class FakeManager : IFakeCallProcessor
     {
-        private static readonly SharedFakeObjectCallRule[] PostUserRules =
+        private static readonly SharedFakeObjectCallRule[] SharedPostUserRules =

```

Comment:
Up until now, all post-user rules were shared, but `EventRule` isn't, so I just made the name more explicit.